File: app/src/main/java/com/example/nickolas/vk/models/enteties/Dialog.java
package com.example.nickolas.vk.models.enteties;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * Created by Nickolas on 04.10.2017.
 */

public class Dialog {
    @SerializedName("unread")
    @Expose
    private Integer unread;
    @SerializedName("message")
    @Expose
    private Message message;
    @SerializedName("in_read")
    @Expose
    private Integer inRead;
    @SerializedName("out_read")
    @Expose
    private Integer outRead;

    public Integer getUnread() {
        return unread;
    }

    public void setUnread(Integer unread) {
        this.unread = unread;
    }

    public Message getMessage() {
        return message;
    }

    public void setMessage(Message message) {
        this.message = message;
    }

    public Integer getInRead() {
        return inRead;
    }

    public void setInRead(Integer inRead) {
        this.inRead = inRead;
    }

    public Integer getOutRead() {
        return outRead;
    }

    public void setOutRead(Integer outRead) {
        this.outRead = outRead;
    }
}

File: app/src/main/java/com/example/nickolas/vk/models/enteties/Dialog.kt
package com.example.nickolas.vk.models.enteties

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName


class Dialog {

    @SerializedName("unread")
    @Expose
    var unread: Int? = null
        get() = if (field == null) 0 else field
    @SerializedName("message")
    @Expose
    var message: Message? = null
    @SerializedName("in_read")
    @Expose
    var inRead: Int? = null
    @SerializedName("out_read")
    @Expose
    var outRead: Int? = null

}